game/kegeland/src/screens/Register/RegistrationQuestionnaire.tsx

```diff
@@ -57,107 +57,103 @@ const RegistrationQuestionnaireScreen = ({
 
   return (
     <Box style={scrollViewStyles.container}>
-      <ScrollView style={scrollViewStyles.scrollView}>
```

Comment:
Since the scrollview is removed, how will this look on a smaller phone? What if the questionnaire and button are getting out of the screen, without possibility to scroll?

Reply:
Wondering about this as well, haven't tried on other phones. If all the sizes and paddings are unitless it should look the same on all phones, but I guess there's a reason that we had scrollview in the first place? On my phone everything fits without scrolling at least